en/config/battery.md

```diff
@@ -164,25 +168,31 @@ This setting corresponds to parameter(s): [BAT1_A_PER_V](../advanced_config/para
 
-PX4 implements a current-based load compensation that uses a real-time estimate of the internal resistance of the battery.
-When a current flows through a battery, the internal resistance causes a voltage drop, reducing the output voltage (measured voltage) of the battery compared to its open-circuit voltage (no-load voltage).
-By estimating the internal resistance, the fluctuation in measured voltage under load that occurs when using the [basic configuration](#basic_settings) can be compensated.
-This leads to a much more accurate estimation of the remaining capacity.
+_Load compensation_ uses a measured or estimated value for the internal resistance to correct for changes under load, resulting in far less variation in the estimated capacity when flying.
+This is enabled by default when using a power module that provides current measurements.
 
-To use the load compensation you will still need to set the [basic configuration](#basic_settings).
+To use the load compensation first set the [basic configuration](#basic_settings).
 The _Empty Voltage_ ([BATn_V_EMPTY](../advanced_config/parameter_reference.md#BAT1_V_EMPTY), where `n` is the battery number) should be set higher (than without compensation) because the compensated voltage gets used for the estimation (typically set a bit below the expected rest cell voltage when empty after use).
-You should also calibrate the [Amps per volt divider](#current_divider) in the basic settings screen.
 
-::: info
-Alternatively, the value for the internal resistance can be [set manually](../advanced_config/parameters.md) using [BAT1_R_INTERNAL](../advanced_config/parameter_reference.md#BAT1_R_INTERNAL) (advanced).
-A positive value in this parameter will be used for the internal resistance instead of the estimated value.
+You will then need to calibrate the [Amps per volt divider](#current_divider) in the basic settings screen (in order to get reliable current measurements).
+
+PX4 uses current-based load compensation based on a _real-time estimate_ of the internal resistance of the battery by default (real time estimates are enabled if [BAT1_R_INTERNAL=-1](../advanced_config/parameter_reference.md#BAT1_R_INTERNAL)).
+Using a real time estimate allows the compensation to adapt to changes in the internal resistance of the battery due to temperature changes during flight, as well as over time as the battery degrades.
+
+The internal resistance can also be measured and [set manually](../advanced_config/parameters.md) in [BAT1_R_INTERNAL](../advanced_config/parameter_reference.md#BAT1_R_INTERNAL).
+A positive value in this parameter will be used for the internal resistance instead of the estimated value (`0` disables load compensation altogether).
+
+::: tip
+The default approach of using the real-time estimate is preferred.
+While in theory this might not converge to a value for the internal resistance, testing has not show this to be a problem.
+:::
+
```

Comment:
```suggestion
A positive value in this parameter will be used for the internal resistance instead of the estimated value (`0` disables load compensation altogether).

```